### ISSUE TITLE
backend_monitor.py U1,U2,U3 error

### ISSUE THREAD

**Issue by hannahaih** [Qiskit/qiskit-terra#5998]
### Information

- **Qiskit Terra version**:
- {'qiskit-terra': '0.16.1',
 'qiskit-aer': '0.7.2',
 'qiskit-ignis': '0.5.1',
 'qiskit-ibmq-provider': '0.11.1',
 'qiskit-aqua': '0.8.1',
 'qiskit': '0.23.2'}

- **Python version**:
- 3.8.5

- **Operating system**:
- Win10 with Anaconda

### What is the current behavior?

`from qiskit import IBMQ`
`IBMQ.load_account()`
`provider = IBMQ.get_provider(hub='ibm-q-community', group='ibmquantumawards', project='open-science')`
`backend = provider.get_backend('ibmq_casablanca')`
`backend`

returns error: 

`---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
C:\ProgramData\Anaconda3\envs\myenv\lib\site-packages\IPython\core\formatters.py in __call__(self, obj)
    339                 pass
    340             else:
--> 341                 return printer(obj)
    342             # Finally look for special method names
    343             method = get_real_method(obj, self.print_method)

C:\ProgramData\Anaconda3\envs\myenv\lib\site-packages\qiskit\tools\jupyter\backend_monitor.py in _backend_monitor(backend)
     99                              'Error Map', 'Job History'])
    100 
--> 101         details.extend([qubits_tab(backend), gates_tab(backend),
    102                         detailed_map(backend), jobs])
    103 

C:\ProgramData\Anaconda3\envs\myenv\lib\site-packages\qiskit\tools\jupyter\backend_monitor.py in qubits_tab(backend)
    306         qubit_html += "<tr><td><font style='font-weight:bold'>%s</font></td><td>%s</td>"
    307         qubit_html += "<td>%s</td><td>%s</td><td>%s</td><td>%s</td><td>%s</td><td>%s</td></tr>"
--> 308         qubit_html = qubit_html % (name, freq, T1, T2, U1, U2, U3, readout_error)
    309     qubit_html += qubit_footer
    310 

NameError: name 'U1' is not defined`

<IBMQBackend('ibmq_casablanca') from IBMQ(hub='ibm-q-community', group='ibmquantumawards', project='open-science')>

### Steps to reproduce the problem

`from qiskit import IBMQ`
`IBMQ.load_account()`
`provider = IBMQ.get_provider(hub='ibm-q-community', group='ibmquantumawards', project='open-science')`
`backend = provider.get_backend('ibmq_casablanca')`
`backend`

### What is the expected behavior?

`backend`

should return:

![image](https://user-images.githubusercontent.com/68157561/110526066-a788f880-8115-11eb-8887-b88790d2b529.png)

### Suggested solutions

in backend_monitor.py add the line:

`         U1, U2, U3 = None, None, None`
`
before line #291

**Comment by taalexander** [comment]
I believe this fixed in the current Qiskit terra master. For me it is working with:

```python
Qiskit Software	Version
Qiskit	0.23.6
Terra	0.17.0
Aer	0.7.5
Ignis	0.5.2
Aqua	0.8.2
IBM Q Provider	0.11.1
System information
Python	3.8.5 (default, Aug 5 2020, 03:39:04) [Clang 10.0.0 ]
OS	Darwin
CPUs	8
Memory (Gb)	32.0
Tue Mar 09 15:31:38 2021 AST

```

**Comment by mtreinish** [comment]
This looks like a duplicate of #5008 which has already been fixed in #5546 and was backpored and included in the [0.16.2 release](https://qiskit.org/documentation/release_notes.html#release-notes-0-16-2-bug-fixes). Can you try upgrading you qiskit version to the latest release (or at least qiskit-terra >=0.16.2) and see if that fixes the issue for you.
